Index: snippets/makefile-mode/.yas-parents
===================================================================
--- snippets/makefile-mode/.yas-parents	(리비전 0)
+++ snippets/makefile-mode/.yas-parents	(리비전 0)
@@ -0,0 +1 @@
+text-mode
\ No newline at end of file
Index: snippets/makefile-mode/makefile
===================================================================
--- snippets/makefile-mode/makefile	(리비전 0)
+++ snippets/makefile-mode/makefile	(리비전 0)
@@ -0,0 +1,26 @@
+#name : Makefile template
+# --
+TARGET = ${1:target program name}
+
+SRCS = ${2:\$(wildcard *.cpp)}
+OBJS = ${3:\$(SRCS:.cpp=.o)}
+
+CXX = ${4:g++}
+CXXFLAGS = ${5:-Wall -g}
+
+$(TARGET) : $(OBJS)
+	$(CXX) $(CXXFLAGS) -o $@ $^
+
+dep :
+	$(CXX) -M $(SRCS) > .depend
+
+%.o : %.cpp
+	$(CXX) $(CXXFLAGS) -c $<
+
+clean:
+	rm -f $(TARGET) $(OBJS) .depend
+
+ifeq (.depend,$(wildcard .depend))
+-include .depend
+endif
+$0
\ No newline at end of file
Index: snippets/cc-mode/do
===================================================================
--- snippets/cc-mode/do	(리비전 659)
+++ snippets/cc-mode/do	(작업 사본)
@@ -1,6 +1,5 @@
 #name : do { ... } while (...)
 # --
-do
-{
+do {
     $0
 } while (${1:condition});
\ No newline at end of file
Index: snippets/cc-mode/for
===================================================================
--- snippets/cc-mode/for	(리비전 659)
+++ snippets/cc-mode/for	(작업 사본)
@@ -1,6 +1,5 @@
 #name : for (...; ...; ...) { ... }
 # --
-for (${1:int i = 0}; ${2:i < N}; ${3:++i})
-{
+for (${1:int i = 0}; ${2:i < N}; ${3:i++}) {
     $0
 }
\ No newline at end of file
Index: snippets/cc-mode/while
===================================================================
--- snippets/cc-mode/while	(리비전 0)
+++ snippets/cc-mode/while	(리비전 0)
@@ -0,0 +1,5 @@
+#name : while (...) { ... }
+# --
+while (${1:condition}) {
+    $0
+}
\ No newline at end of file
Index: snippets/cc-mode/if
===================================================================
--- snippets/cc-mode/if	(리비전 659)
+++ snippets/cc-mode/if	(작업 사본)
@@ -1,6 +1,5 @@
 #name : if (...) { ... }
 # --
-if (${1:condition})
-{
+if (${1:condition}) {
     $0
 }
\ No newline at end of file
